openshift.spec

```diff
@@ -99,6 +99,81 @@ Requires:         ethtool
 %description sdn-ovs
 %{summary}
 
+
+# ====
+%package -n atomic-enterprise
+Summary:         Open Source Platform as a Service by Red Hat
+
+
+%description -n atomic-enterprise
+%{summary}
+
+
+%package -n atomic-enterprise-master
+Summary:        Atomic Enterprise Master
+Requires:       atomic-enterprise = %{version}-%{release}
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
+
+%description -n atomic-enterprise-master
+%{summary}
+
+%package -n atomic-enterprise-node
+Summary:        Origin Node
+Requires:       atomic-enterprise = %{version}-%{release}
+Requires:       docker-io >= 1.6.2
```

Comment:
@miminar by "all these versions" do you mean requires versions (like the docker-io above)?